src/convertToObject.js

```diff
@@ -6,7 +6,44 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splitArray = sourceString.trim().split(';');
```

Comment:
The variable `splitArray` is named in a way that could decrease code readability as it is too generic. Consider a more descriptive name that reflects its contents, such as `styleDeclarations`.